pkg/model/manifest.go

```diff
@@ -263,6 +263,51 @@ func NewDeployInfo() *DeployInfo {
 	}
 }
 
+//GetManifestV1 gets a manifest from a path or search for the files to generate it
+func GetManifestV1(manifestPath string) (*Manifest, error) {
+	cwd, err := os.Getwd()
+	if err != nil {
+		return nil, err
+	}
+	if manifestPath != "" && !filepath.IsAbs(manifestPath) {
+		manifestPath = filepath.Join(cwd, manifestPath)
+	}
+	if manifestPath != "" && FileExistsAndNotDir(manifestPath) {
+		manifest, err := getManifestFromFile(cwd, manifestPath)
+		if err != nil {
+			return nil, err
+		}
+		path := ""
+		if filepath.IsAbs(manifestPath) {
+			path, err = filepath.Rel(cwd, manifestPath)
+			if err != nil {
+				oktetoLog.Debugf("could not detect relative path to %s: %s", manifestPath, err)
+			}
+		}
+		manifest.Filename = path
+		return manifest, nil
+	} else if manifestPath != "" && pathExistsAndDir(manifestPath) {
+		cwd = manifestPath
+	}
+
+	var devManifest *Manifest
+	if oktetoPath := getFilePath(cwd, oktetoFiles); oktetoPath != "" {
```

Comment:
We already have this logic in `GetManifestV2`. Can we called it `getManifestFromOktetoFile`